### PR TITLE
Touch typing

### PR DESCRIPTION
**Pull request by lucax88x**
some corrections and improvements

### CODE REVIEW

File: _posts/2016-12-07-forget-the-keyboard.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ If you are learning to touch type, use this cheat sheet. Print it, and forget th
 
 # It's about your fingers, not about the keyboard
 
-I'm a late touch typer. I learned to type without looking in my 40s, after years of bad habits. Consequently, learning to touch type hasn't be the simplest task to me.
+I'm a late touch typer. I learned to type without looking in my 40s, after years of bad habits. Consequently, learning to touch type hasn't been the simplest task to me.
```
Comment (arialdomartini) [inline _posts/2016-12-07-forget-the-keyboard.md]: Thank you!

File: _posts/2016-12-07-forget-the-keyboard.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ When your fingers are on the home row, you know that if you type with *that* fin
 
 ![Position of letter j](static/img/hands-keyboard-j.jpg)
 
-Well, that's all you need to know  about **`J`**. Do you mind *where* the **`J`** key is, on the keyboard? No, you don't. Just don't think to the physical key under your finger. **Start thinking that your right hand index is the `J` letter**. As long you don't move your hands from the home row, that finger will always be the **`J`**. It's a simple association that your muscle memory will easily remember.
+Well, that's all you need to know  about **`J`**. Do you mind *where* the **`J`** key is, on the keyboard? No, you don't. Just don't think to the physical key under your finger. <br/>**Start thinking that your right hand index is the `J` letter**. As long you don't move your hands from the home row, that finger will always be the **`J`**. It's a simple association that your muscle memory will easily remember.
```
Comment (arialdomartini) [inline _posts/2016-12-07-forget-the-keyboard.md]: Ok, I like to put this in a new line, but still in the same paragraph. Good hint, thank you.

File: _posts/2016-12-07-forget-the-keyboard.md [hunk 3]
```diff
@@ -22,6 +22,10 @@ Besides the unavoidable and boring hours of exercise, the trick that helped me t
 >
 > Only focus on *which finger to use*.
 
+*What do you mean?*
```
Comment (arialdomartini) [inline _posts/2016-12-07-forget-the-keyboard.md]: Not convinced. 

TL;DR stands for "Too long, don't read", and it is intended to be like a sort of abstract. Just like the "occhiello" o "sommario" in an article.

I would remove that.

Proposal: we could use a different typeset. See for example this [article](https://blog.medium.com/blogging-on-medium-95f1546bcd7d#.4yv4yam3a) (I just took a random one). It uses *italic* font to distinguish the TL;DR section from the rest of the post.

I'm merging this PR, and later I think I will revert this commit, ok?

File: _posts/2016-12-07-forget-the-keyboard.md [hunk 4]
```diff
@@ -49,7 +49,8 @@ When your fingers are on the home row, you know that if you type with *that* fin
 
 ![Position of letter j](static/img/hands-keyboard-j.jpg)
 
-Well, that's all you need to know  about **`J`**. Do you mind *where* the **`J`** key is, on the keyboard? No, you don't. Just don't think to the physical key under your finger. <br/>**Start thinking that your right hand index is the `J` letter**. As long you don't move your hands from the home row, that finger will always be the **`J`**. It's a simple association that your muscle memory will easily remember.
+Well, that's all you need to know  about **`J`**. Do you mind *where* the **`J`** key is, on the keyboard? No, you don't. Just don't think to the physical key under your finger. <br/>
```
Comment (arialdomartini) [inline _posts/2016-12-07-forget-the-keyboard.md]: As stated before, I like to have this sentence emphasized in a new line, but I keep thinking it belongs to the same paragraph. I'm reverting this commit.

File: _posts/2016-12-07-forget-the-keyboard.md [hunk 5]
```diff
@@ -56,7 +56,7 @@ Equally, for getting a **`U`** you have to move your right hand index up.
 
 ![Position of letter j](static/img/hands-keyboard-ju.jpg)
 
-Use the trick again: **there's no physical `U` key, it's your finger that has a `U` letter on its tip**. The cheat sheet can help you visualize it. You don't need to look or think to a keyboard to make it work.
+Use the trick again: **there's no physical `U` key, it's your finger that has a `U` letter on its tip**.
```
Comment (arialdomartini) [inline _posts/2016-12-07-forget-the-keyboard.md]: You are completely right. Thank you for helping me keeping the post shorter.
